larky-api/src/main/java/com/verygood/security/larkyapi/VersionedLarkyEngineImpl.java

```diff
@@ -0,0 +1,214 @@
+package com.verygood.security.larkyapi;
+
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.support.PathMatchingResourcePatternResolver;
+
+import javax.annotation.Nonnull;
+import javax.script.*;
```

Comment:
Please, avoid using star imports, it clutters the local namespace.

Reply:
done